### ISSUE TITLE
Establish extent data fits or differs with that produced by/for the UN on 11.2.1

### ISSUE THREAD

**Issue by james-westwood** [ONSdigital/SDG_11.2.1#175]
[quartm](https://github.com/quartm) asked:
"To what extent does the data you’ve produced fit it or differ with the produced by the UN at the city level for various UK urban areas?"
https://data.unhabitat.org/datasets/GUO-UN-Habitat::11-2-1-percentage-access-to-public-transport/explore?showTable=true

To answer this we need to look at the sources of the other datasets and understand:
- The methodology for the calculation used
- How granular their data is for population centres. (Ours is at OA level)
- The age groups they have used
- Their data sources and definitions, in particular:
    - Definition of "Urban" and "Rural"
    - Definition of "Disabled" and "Non-disabled"

There may be other points to consider for a good comparison

**Comment by quartm** [comment]
Hi James, 

I've found the source of the data that is provided for the UK (produced by the EU and supplied to the UN): https://ec.europa.eu/regional_policy/en/information/publications/working-papers/2020/low-carbon-urban-accessibility

At the bottom of this page they have included the methodology and python scripts that might be worth a look. There is also a workflow description which describes each step in the code in a succinct document.
[SDG_11_2_1_REGIO_technical_document_v2.docx](https://github.com/ONSdigital/SDG_11.2.1/files/8089123/SDG_11_2_1_REGIO_technical_document_v2.docx)

**Comment by jwestw** [comment]
Thank you for the links @quartm 

I have looked through the Python scripts. Their approach is really quite different scripting-wise, and they have relied on a geospatial programme like Qgis, so I am finding it a bit hard to follow. What's really good though is their method write up. Essentially it's the same steps we are following:


[SDG_11_2_1_REGIO_technical_document_v2.docx](https://github.com/ONSdigital/SDG_11.2.1/files/8142739/SDG_11_2_1_REGIO_technical_document_v2.docx)

The one step we were missing was that which takes into account transport capacity, however this will now be covered by:  https://github.com/ONSdigital/SDG_11.2.1/issues/177

**Comment by quartm** [comment]
@jwestw Glad the write-up was useful and its definitely good news that your method aligns well!

**Comment by jwestw** [comment]
To do:

Make a document to outline the similarities and differences between the UN/EU code and method and our method based on UN-SDG methodology.

**Comment by james-westwood** [comment]
@Musa-chirikeni
I am trying to find a definition of Rural and Urban for this write up. Could you please help.
Resources I have found so far include:

- [Defining rural areas (2017)](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/597751/Defining_rural_areas__Mar_2017_.pdf)
- [The 2011 Rural-Urban Classification for Output Areas in England](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1009128/RUCOA_leaflet_Jan2017.pdf)
- [The 2011 Rural-Urban Classification For
Small Area Geographies](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/239478/RUC11user_guide_28_Aug.pdf)


From the leaflet, I got this:
Urban areas are the connected built up areas identified by
Ordnance Survey mapping that have resident populations
above 10,000 people (2011 Census).
- Rural areas are those areas that are not urban, i.e.
consisting of settlements below 10,000 people or are open
countryside.
 - For the 2011 Census, England was divided into 171,372
Output Areas (OAs) which on average have a resident
population of 309 people. OAs are the smallest geographic
unit for which Census data are available. Their
geographical size will vary depending on the population
density.

**Comment by james-westwood** [comment]
Write up so far: https://docs.google.com/document/d/1WZUjNgiQwbN0AOrqpznFUNeYeo_XncuIzyAFFOm4LOA/edit?usp=sharing

**Comment by james-westwood** [comment]
@Antonio-John 

Could you make a review of the document linked above please? There are a few highlighted points that we need to discuss. The one about trams/metro/underground links back the the capacity issue you were working on, and should be resolved before we finalise this write-up I think.

**Comment by paigeh1** [comment]
@jwestw I think you have a write-up here but this hasn't been converted to markdown
